checks/check_extra7149

```diff
@@ -31,7 +31,7 @@ extra7149() {
             for redshiftcluster in $LIST_OF_REDSHIFT_CLUSTERS; do
                 REDSHIFT_SNAPSHOT_ENABLED=$($AWSCLI redshift describe-cluster-snapshots $PROFILE_OPT --region $regx --cluster-identifier $redshiftcluster --snapshot-type automated)
                 if [[ $REDSHIFT_SNAPSHOT_ENABLED ]]; then
-                    textPass "$regx: Redshift cluster $redshiftcluster has automated snapshots $REDSHIFT_SNAPSHOT_ENABLED" "$regx" "$redshiftcluster"
+                    textPass "$regx: Redshift cluster $redshiftcluster has automated snapshots." "$regx" "$redshiftcluster"
```

Comment:
why this change?

Reply:
REDSHIFT_SNAPSHOT_ENABLED is a json output. It does not format well, contains commas and multi-line.